packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -351,21 +349,21 @@ class _SnackBarState extends State<SnackBar> {
   @override
   void initState() {
     super.initState();
-    widget.animation.addStatusListener(_onAnimationStatusChanged);
+    widget.animation!.addStatusListener(_onAnimationStatusChanged);
```

Comment:
What's guaranteeing here that `animation` is not null?

Reply:
Similar question everywhere below.

Reply:
Nothing, but it works as before :p More seriously there's an assert non-null in build() at L387 but I really don't know how it works. Trying to make animation required cause a lot of warning. Perhaps @HansMuller could take a look?

Reply:
I see, thanks.